homeassistant/components/wled/light.py

```diff
@@ -139,7 +140,7 @@ async def async_turn_off(self, **kwargs: Any) -> None:
     @wled_exception_handler
     async def async_turn_on(self, **kwargs: Any) -> None:
         """Turn on the light."""
-        data = {ATTR_ON: True}
+        data: dict[str, Any] = {ATTR_ON: True}
```

Comment:
`Any` should be used here, as it is perfectly clear which types it could contain.